### ISSUE TITLE
Moment of inertia

### ISSUE THREAD

**Issue by annetbr** [IEAWindTask37/IEA-10.0-198-RWT#8]
Hi, 

Does anyone have a value of the total moment of inertia of the blades for the IEA 10 MW reference turbine? 

Thank you in advance, 

Annette

**Comment by ptrbortolotti** [comment]
Dear Annette,

I'm terribly sorry for the embarrassing delay addressing your question. You probably have found your solution already, but just in case I attach here the overall mass moments of inertia of all blades in yaw coordinate system

Ixx = 2.24016659e+08
Iyy = 1.07002115e+08
Izz = 8.40464128e+07
Ixy = Ixz = Iyz = 0.

![](http://wisdem.github.io/CCBlade/_images/wind_yaw.png)

If you're interested in the mass moments of inertia along blade span, please check 
https://github.com/IEAWindTask37/IEA-10.0-198-RWT/issues/19

These numbers were computed using WISDEM v3.0.0 by running PreComp.

I'll keep the issue open for a few more weeks in case you have any doubt.

Best regards,
Pietro

**Comment by eletroinf** [comment]
Hello!
Just a confirmation.  Ixx = 2.24016659e+08 should be the Rotor Inertia?
I know that OpenFAST prompts the rotor Inertia in one of the generated .sum files, but I need this paramenter to a simplified model and I don't want to run this model in OpenFAST just to get this parameter. For IEA 15 MW model I found the value of the rotor inertial in OpenFAST yaml files, but this one for 10 MW I did not found it.

Regards
Ricardo
